### DISCUSSION TITLE
Repeated disconnect/reconnect

### DISCUSSION THREAD

**Discussion by MikeMillerDevices** [dotnet/MQTTnet#1909] [General]
I'm connected to a solace broker and see repeatedly that there is a disconnect followed by an immediate reconnect. I don't see any errors or exceptions (all managed enabled). The is no information in the disconnect other than it was a normal connection. 

This doesn't seem to happen all the time but it's more often than not. It's also happens fast than the configured reconnect interval

I'm using a basic username/password authentication.

**Comment by justinW859** [comment]
I've seen this happen when two connections have the same ClientId. They almost fight over the same connection. Note, I'm not a maintainer just a user not an expert.

**Reply by SeppPenner** [reply]
This can indeed be a problem.
